src/containers/Tenant/Query/QueryEditorControls/EditorButton.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../../../styles/mixins.scss';
+
+.ydb-query-editor-button {
+    &__explain-button,
+    &__stop-button,
+    &__run-button {
+        width: 92px;
+    }
+
+    &__stop-button {
+        color: var(--g-color-text-brand-contrast);
+        background-color: var(--g-color-text-danger);
```

Comment:
Should we really make `Stop` button so dangerous? Maybe `view=outlined-danger` will be enough?

Reply:
talked to Alina - made it Action